xml_converter/src/xml_converter.cpp

```diff
@@ -37,10 +32,36 @@ bool has_suffix(std::string const &fullString, std::string const &ending) {
     }
 }
 
+void write_xml_file(string xml_filepath, map<string, Category>* marker_categories, vector<Parseable*>* parsed_pois) {
+    ofstream outfile;
+    string tab_string;
+    string file_text;
+    string new_file_path = xml_filepath + "export.xml";
+
+    file_text = "<OverlayData>\n";
+    for (const auto & category : *marker_categories) {
+        string text;
+        for (const auto& s : category.second.as_xml()) { text += s; };
+        file_text = file_text + text + "\n";
+    }
+
+    file_text = file_text + "<POIs>\n";
+    for (const auto & parsed_poi : *parsed_pois) {
+        string text; 
+        for (const auto& s : parsed_poi->as_xml()) { text += s; };
```

Comment:
Break this out to have the inner loop on a new line

Reply:
Done